### PR TITLE
[Excel] (Custom functions) Adds _ mention in autogen article

### PR DESCRIPTION
**Pull request by mscharlock**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[3ae9cd5](https://github.com/OfficeDev/office-js-docs-pr/pull/1036/commits/3ae9cd5bae6f8822b0421197e3c9984f2349bd28)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-json-autogeneration.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-fixid/docs/excel/custom-functions-json-autogeneration.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C24%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905241850073799-1036%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=eweeY2dNdOaNhPbglprEhWv3Jt76ehXalRM%2Bxw%2BdOus%3D&st=2019-05-24T18%3A46%3A20Z&se=2019-06-24T18%3A51%3A20Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by mscharlock** [comment]
So it turns out this is possibly a situation where SL is unparallel to CFs and we need to figure out what ENG wants. I'll leave this here for the moment but do not merge until confirm with ENG.

File: docs/excel/custom-functions-json-autogeneration.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ The id is used as the invariant identifier for the custom function stored in the
 
 * If id is not provided, the JavaScript/TypeScript function name is converted to uppercase, disallowed characters are removed.
```
Comment (ElizabethSamuel-MSFT) [inline docs/excel/custom-functions-json-autogeneration.md]:
```suggestion
* If id is not provided, the JavaScript/TypeScript function name is converted to uppercase and disallowed characters are removed.
```

File: docs/excel/custom-functions-json-autogeneration.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ The id is used as the invariant identifier for the custom function stored in the
...
 * The id must be unique for all custom functions.
-* The characters allowed are limited to: A-Z, a-z, 0-9, and period (.).
+* The characters allowed are limited to: A-Z, a-z, 0-9, underscores (\_), and period (.).
```
Comment (ElizabethSamuel-MSFT) [inline docs/excel/custom-functions-json-autogeneration.md]:
```suggestion
* The allowed characters are limited to: A-Z, a-z, 0-9, underscores (\_), and period (.).
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[7da8451](https://github.com/OfficeDev/office-js-docs-pr/pull/1036/commits/7da84510edbbca05b075fbe1bdbbab9ebe8f4bae)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-json-autogeneration.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-fixid/docs/excel/custom-functions-json-autogeneration.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C30%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905301837540785-1036%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=XDqM6ModawUt1kL6GB2CLF9csr%2Fr3hMQQcLQMrXMdg0%3D&st=2019-05-30T18%3A34%3A02Z&se=2019-06-30T18%3A39%3A02Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[c3ea776](https://github.com/OfficeDev/office-js-docs-pr/pull/1036/commits/c3ea776e123329a707c68e9634d744725469b211)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-a8d76259)

<a id="a8d76259"></a>
###
 - **[Error]** 
`````
Cannot sync git repo to specified commit because branch michelle-fixid has been deleted or has been force pushed: fatal: Couldn't find remote ref michelle-fixid

`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C30%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905301838081216-1036%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=NbSaTczpxJAar5r%2FsJ7E8kFhbwXh1RehYz5BxDr4GkI%3D&st=2019-05-30T18%3A34%3A23Z&se=2019-06-30T18%3A39%3A23Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**